### PR TITLE
remove a duplicate file

### PR DESCRIPTION
**Pull request by whitewum**
1. fix some minor errors

2. remove a duplicate file

### CODE REVIEW

File: docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md [hunk 1]
```diff
@@ -29,6 +29,8 @@
 * Linux 内核：3.9 或更高版本，通过命令 `uname -r` 查看
 * glibc：2.12 或更高版本，通过命令 `ldd --version` 查看
 
+操作系统配置见[这里](./7.kernel-config.md)
```
Comment (Amber1990Zhang) [inline docs/manual-CN/3.build-develop-and-administration/3.configurations/0.system-requirement.md:32]: add 。at the end

File: docs/manual-CN/3.build-develop-and-administration/3.configurations/1.config-persistency-and-priority.md [hunk 2]
```diff
@@ -15,4 +15,4 @@
 对于一个参数项：
 
 - 默认的配置寻找优先级：meta service > 命令行 `UPDATE CONFIG` 语法 > 本地环境变量 > 本地配置文件。
-- 如果将 `--local_config` 设置为 true，则先到本地配置文件中寻找；然后依次 meta service, 和环境变量。
+- 如果 `--local_config=true`：本地配置文件 > meta service > 环境变量。
```
Comment (Amber1990Zhang) [inline docs/manual-CN/3.build-develop-and-administration/3.configurations/1.config-persistency-and-priority.md:18]: 环境变量 >- 本地环境变量
please make your expression unified.

File: docs/manual-CN/3.build-develop-and-administration/3.configurations/2.configs-syntax.md [hunk 3]
```diff
@@ -116,7 +43,7 @@ nebula> GET CONFIGS storage:v;
 UPDATE CONFIGS [graph|storage :] <var> = <value>
 ```
 
-> 更新的变量将持久化在 meta-service 中。
+> 更新的变量将持久化在 `meta service` 中。
```
Comment (Amber1990Zhang) [inline docs/manual-CN/3.build-develop-and-administration/3.configurations/2.configs-syntax.md:46]: why deleted the introduction? I think we only delete the `gflag` section.

File: docs/manual-CN/README.md [hunk 4]
```diff
@@ -118,7 +118,7 @@
   * [Graphd 配置](3.build-develop-and-administration/3.configurations/4.graph-config.md)
   * [Storaged 配置](3.build-develop-and-administration/3.configurations/5.storage-config.md)
   * [命令行终端配置](3.build-develop-and-administration/3.configurations/6.console-config.md)
-  * [Kernel 配置](3.build-develop-and-administration/3.configurations/6.console-config.md)
+  * [Kernel 配置](3.build-develop-and-administration/3.configurations/7.kernel-config.md)
```
Comment (Amber1990Zhang) [inline docs/manual-CN/README.md:121]: if you change the file name, please update the file name [here](https://github.com/vesoft-inc/nebula-docs-cn/blob/master/mkdocs.yml) in the `yaml` file.
image

File: docs/manual-CN/3.build-develop-and-administration/3.configurations/2.configs-syntax.md (hunk 3 above)
Comment (whitewum) [reply docs/manual-CN/3.build-develop-and-administration/3.configurations/2.configs-syntax.md:46]: I think the introduction is duplicate.

**Pull request merged**